### DISCUSSION TITLE
How is passkit-generator helping your business?

### DISCUSSION THREAD

**Discussion by alexandercerutti** [alexandercerutti/passkit-generator#243] [General]
Hi everyone!

Passkit-generator has reached an astonishing level of popularity that I would have never imagined when I started it over 6 years ago. Right now it seems (might be wrong tho) the most liked and appreciated project for creating Apple Wallet passes on demand.

It has been cited on the following resources:

 - [Nearform blog post](https://www.nearform.com/blog/how-to-create-custom-apple-wallet-passes-with-react-native-and-fastify/)
 - [Youtube video by scriptpapi](https://www.youtube.com/watch?v=rJZdPoXHtzI)
 - [Ionic documentation for Apple Wallet passes](https://ionic.io/docs/tutorials/integrations/digitial-passes/apple-wallet-passes/creating)
 - [Italia Open Source](https://italiaopensource.com/opensources/passkit-generator)
 - Several blog posts, Medium articles and reddit pages.
 
Furthermore, according to past insights I saw from stats of this repo and interactions, `passkit-generator` might be being used by several famous companies around the world (nothing official tho), like Flixbus, Audi, MOMA, [Standford Alumni](https://github.com/SU-SWS/saa_alumni) and many more I don't remember, as well as several little business all around the world.

It has around ~16.000 - 22.000~ 30.000 - 34.000 downloads per week (not unique, ofc) and over ~800~ 1000 🌟 on Github.

And I'd like to thank you for the trust you put in such project.

I'd like this project to reach the point of which I could provide support to people to integrate advanced features like NFC, iCloud Binding and so on.

If you could share a statement about how this project helped you, that would be awesome.

Thank you very much ❤️
Alexander

**Comment by kimidion** [comment]
I just started using your library and it's working well so far. I am partnered with an attraction for tickets and passholder distribution (no actual revenue from it yet, but hopefully soon!). I will be looking into obtaining NFC certification in the future when I can get the business on board with getting combo NFC & barcode scanning terminals. I'd be happy to collaborate and provide testing feedback if I am granted access.

**Comment by qodifynl** [comment]
Hi Alexander, 

Please contact me as we can work on a collaboration regarding NFC. Please send me a message on info@fifty2.ae

**Comment by alexandercerutti** [comment]
Hi @qodifynl, can you contact me on Telegram?

**Comment by qodifynl** [comment]
Please send a message to 'oktaydxb'

**Comment by jannis6023** [comment]
I guess Apple will *never* issue a certificate to you - at least not as long as you are developing this library. And even if you would ever be granted access to one officially, there would be a restrictive NDA.
I think that it's soo sad that they hide their great technology behind all that, too.
Maybe this helps: I found out, that they released a large page with documentation concerning the access stuff: https://register.apple.com/resources/docs/apple-pay/access/program-guide/

It is located on the wallet information site in the last box.

I really love this library and we use it at https://edutix.de (no ad here - just showing the use case) for generating Wallet passes for graduation proms. The students love it as well as we did last year at our proms ourselves.

**Comment by alexandercerutti** [comment]
Thank you @jannis6023!

I gave the link a fast read. I wasn't aware of it. Some pages and documents appear to be very well hidden.

Ofc, if these are the requirements (a.k.a. "the famous undisclosed requirements" - but I think there are more), I will never be able to obtain one (even if by signing an NDA).

**Comment by SoCuul** [comment]
What are the requirements to be able to obtain the NFC certificates? If I have a registered business and want to experiment with using NFC passes for internal scanning, would that be enough to get approved?

**Comment by alexandercerutti** [comment]
> What are the requirements to be able to obtain the NFC certificates? If I have a registered business and want to experiment with using NFC passes for internal scanning, would that be enough to get approved?

@SoCuul I think you need to check the link that has been provided above, that specifies all the company structure requirements.

They say such requirements are undisclosed, so I guess there is more actually.
